ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -37,6 +39,11 @@ const FormFieldsContainer = () => {
     []
   );
 
+  const { values, setFieldValue, setFieldTouched } = useFormikContext();
+
+  const validTags = "strong,b,div,br,p,i,li";
```

Comment:
Is there any reason to not to use the default accepted tags set, consistent with what backend also allows?